packages/splitter/src/useSplitter.ts

```diff
@@ -25,18 +30,24 @@ interface IEventTargetLike {
 
 interface IBodyLike extends IEventTargetLike {
   clientWidth: number;
+  clientHeight: number;
 }
 
 interface IDocumentLike extends IEventTargetLike {
   body: IBodyLike;
 }
+
+export interface IWindowLike {
+  document: IDocumentLike;
+  scrollX: number;
+  scrollY: number;
+}
+
 export interface IUseSplitterProps extends Omit<HTMLProps<any>, 'onChange'> {
   /** An aria-label for the separator */
   ariaLabel?: string;
-  /** An id of the element the separator controls */
-  primaryPaneId: string;
-  /** A document environment to attach events to */
-  environment?: Document | IDocumentLike;
+  /** A window environment to attach events to */
+  environment?: Window | IWindowLike;
```

Comment:
Needed `window` in order to grab `scrollX` and `scrollY` for offset calculation below.

Reply:
What other options were considered for `window`? Here's another situation where `window` is needed in a SSR context: https://github.com/zendeskgarden/website/blob/main/src/layouts/Root/components/SearchInput.tsx

Reply:
Do other DOM objects contain the global `scrollX` and `scrollY`? Im not sure what you mean. Does Garden conventions allow for global `window` usage without a proxy `environment` prop in light of SSR? If so, I'll refactor things back to `document`.